model/riscv_insts_zicbom.sail

```diff
@@ -78,6 +78,7 @@ function process_clean_inval(rs1, cbop) = {
   match ext_data_get_addr(rs1, negative_offset, Read(Data), cache_block_size) {
     Ext_DataAddr_Error(e) => Ext_DataAddr_Check_Failure(e),
     Ext_DataAddr_OK(vaddr) => {
+      let vaddr = transform_effective_address(vaddr, Read(Data));
```

Comment:
Why can't we have the pointer masking be handled by ext_data_get_addr()? That hook allows checking+transforming a data memory access and seems like the ideal place to add the pointer masking hooks?